src/JSConfetti.ts

```diff
@@ -78,8 +128,10 @@ class JSConfetti {
       y: yPosition,
     }
 
+    const confettiGroup = new ConfettiBatch()
+
     for (let i = 0; i < confettiNumber / 2; i++) {
-      this.shapes.push(new ConfettiShape({
+      const right = new ConfettiShape({
```

Comment:
maybe we can call them `confettiesOnTheRight` & `confettiesOnTheLeft` so the code is more readable?

Reply:
Yep! Done!